### DISCUSSION TITLE
Who is the only person to win an 🎥🏆 playing another real 🎥🏆 winner?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#976] [Galaxy Trivia] [answered]
**Bonus:** For a role in which movie?

**Comment by peterbaccaro** [comment, marked as answer]
The only person to win a movie Oscar for playing another real movie Oscar winner is **Cate Blanchett** for the role of **Katherine Hepburn** in the movie **The Aviator**.
